reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+
+See https://discourse.ubuntu.com/t/enable-high-availability/17754.
+
+<a name="hook"></a>
+### Hook
+
+Code that is invoked at different points in time in the life cycle of a container. Hooks are part of addons.
+
+See [Hooks](https://discourse.ubuntu.com/t/addons/25293#hooks).
+
+<a name="image"></a>
+### Image
+
+The base for a container, which contains all necessary components like Anbox or the Android root file system. Anbox Cloud provides images based on different Android versions and different architectures.
+
+See https://discourse.ubuntu.com/t/manage-images/17758 and https://discourse.ubuntu.com/t/provided-images/24185.
+
+<a name="instance-type"></a>
+### Instance type
+
+An abstraction for a set of resources that is available to a container.
+
+See https://discourse.ubuntu.com/t/instance-types/17764.
+
+<a name="juju"></a>
+### Juju
+
+A charmed operator framework that helps you deploy, integrate and manage applications across multiple environments. Anbox Cloud is installed using Juju. The Anbox Cloud Appliance uses Juju under the hood.
+
+See [the Juju website](https://juju.is/).
+
+<a name="lxd"></a>
+### LXD
+
+A system container and virtual machine manager that offers a unified user experience around full Linux systems running inside containers or virtual machines. Anbox Cloud is based on LXD.
+
+See [the LXD website](https://linuxcontainers.org/).
+
+<a name="monitoring-stack"></a>
+### Monitoring stack
+
+The part of the Anbox Cloud stack that provides monitoring functionality.
+
+See https://discourse.ubuntu.com/t/install-the-monitoring-stack/17786.
+
+<a name="nagios"></a>
+### Nagios
+
+A tool for monitoring the status of critical infrastructure like networks, servers and applications.
+
+See [the Nagios website](https://www.nagios.org/).
+
+<a name="nats"></a>
+### Neural Autonomic Transport System (NATS)
+
+An open-source messaging system that the components of the streaming stack use to communicate.
+
+See [the NATS website](https://nats.io/).
+
+<a name="platform"></a>
+### Platform
+
+An abstraction layer that is provided by Anbox to access the hardware resources of the host system from the Android system. Anbox Cloud supports three platforms: null (without rendering), webrtc (WebRTC) and swrast (software rendering).
+
+See https://discourse.ubuntu.com/t/anbox-platforms/18733.
+
+<a name="prometheus"></a>
+### Prometheus
+
+An open-source application used for event monitoring and alerting, which records real-time metrics about system events.
+
+See [the Prometheus website](https://prometheus.io/).
+
+<a name="raw-container"></a>
+### Raw container
+
+A container that is created when launching an image. It runs the full Android system, without any additional apps installed.
+
+See [Application containers vs. raw containers](https://discourse.ubuntu.com/t/about-containers/17763#application-vs-raw).
+
+<a name="regular-container"></a>
+### Regular container
```

Comment:
Do we need to explain this ? I feel like it just brings more confusion, it's the same as the definition for `container`

Reply:
Currently we are making a distinction between regular containers and base containers, both in the docs and in the container type. Can we get rid of that distinction?

Reply:
Not really, we have to make a distinction between a base container and a regular container. However in the context of the glossary, `container` and `regular container` are the same type of containers
Although it makes sense to specify both, considering `container` is the generic term and `regular container` is just there to contrast the definition of `base container`
After consideration, let's leave it as is